packages/react-component-library/src/components/Timeline/TimelineRow.tsx

```diff
@@ -41,7 +41,7 @@ export interface TimelineRowProps extends ComponentWithClass {
   /**
    * Supply a custom presentation layer.
```

Comment:
Think it would be good to update the docs here to make it clear what is being rendered (the row header if `hasSide` is set I think).